api-tests/src/interfaces/process-cri-callback-response.ts

```diff
@@ -0,0 +1,3 @@
+export interface ProcessCriCallbackResponse {
```

Comment:
I think it technically returns a `JourneyEngineResponse` - I think if there's an error it'll return a page id rather than a journey step.